ext/ftp/ftp.stub.php

```diff
@@ -57,48 +57,48 @@ function ftp_systype($ftp): string|false {}
 
 /**
  * @param resource $ftp
- * @param resource $fp
+ * @param resource $stream
  */
-function ftp_fget($ftp, $fp, string $remote_file, int $mode = FTP_BINARY, int $resumepos = 0): bool {}
+function ftp_fget($ftp, $stream, string $remote_filename, int $mode = FTP_BINARY, int $offset = 0): bool {}
 
 /**
  * @param resource $ftp
- * @param resource $fp
+ * @param resource $stream
  */
-function ftp_nb_fget($ftp, $fp, string $remote_file, int $mode = FTP_BINARY, int $resumpos = 0): int {}
+function ftp_nb_fget($ftp, $stream, string $filename, int $mode = FTP_BINARY, int $offset = 0): int {}
```

Comment:
This one is inconsistent with ftp_fget.

Reply:
Oh, no. Hopefully, my newly amended commit manages to address all the issues this time.